### PR TITLE
OM-540 | Timmi connect to berth

### PR DESCRIPTION
**Pull request by pjlampil**
- Add service `BERTH` for imported profiles
- Remove `import Profile` from `Service` to avoid circular imports
- Fix reformats

### CODE REVIEW

File: profiles/models.py [hunk 1]
```diff
@@ -179,6 +181,11 @@ def import_customer_data(cls, data):
                     profile.phones.create(
                         phone=phone, phone_type=PhoneType.MOBILE, primary=index == 0
                     )
+                ServiceConnection.objects.create(
+                    profile=profile,
+                    service=Service.objects.get(service_type=ServiceType.BERTH),
```
Comment (igordavydsson) [inline profiles/models.py]: Is it a big task to make the `Service` as dropdown in the import form?

Comment (igordavydsson) [reply profiles/models.py]: not sure how necessary that is, but I think the import could be used by any service, at least in theory

Comment (pjlampil) [reply profiles/models.py]: Not really a huge task, but I would leave it as part of the story where we generalize the whole import. Now this was strictly scoped as `Timmi => Venepaikka` 🤓

Comment (igordavydsson) [reply profiles/models.py]: ok, fair enough 👍

**Pull request merged**